src/EnergyPlus/UnitarySystem.cc

```diff
@@ -106,6 +106,22 @@ namespace UnitarySystems {
     static std::string const fluidNameSteam("STEAM");
     static std::string const blankString("");
 
+    UnitarySysInputSpec::UnitarySysInputSpec()
+        : dx_heating_coil_sizing_ratio(1.0), minimum_supply_air_temperature(2.0), cooling_supply_air_flow_rate(-999.0),
+          cooling_supply_air_flow_rate_per_floor_area(-999.0), cooling_fraction_of_autosized_cooling_supply_air_flow_rate(-999.0),
+          cooling_supply_air_flow_rate_per_unit_of_capacity(-999.0), heating_supply_air_flow_rate(-999.0),
+          heating_supply_air_flow_rate_per_floor_area(-999.0), heating_fraction_of_autosized_heating_supply_air_flow_rate(-999.0),
+          heating_supply_air_flow_rate_per_unit_of_capacity(-999.0), no_load_supply_air_flow_rate(-999.0),
+          no_load_supply_air_flow_rate_per_floor_area(-999.0), no_load_fraction_of_autosized_cooling_supply_air_flow_rate(-999.0),
+          no_load_fraction_of_autosized_heating_supply_air_flow_rate(-999.0),
+          no_load_supply_air_flow_rate_per_unit_of_capacity_during_cooling_operation(-999.0),
+          no_load_supply_air_flow_rate_per_unit_of_capacity_during_heating_operation(-999.0), maximum_supply_air_temperature(80.0),
+          maximum_outdoor_dry_bulb_temperature_for_supplemental_heater_operation(21.0), maximum_cycling_rate(2.5), heat_pump_time_constant(60.0),
+          fraction_of_on_cycle_power_use(0.01), heat_pump_fan_delay_time(60.0), ancillary_on_cycle_electric_power(0.0),
+          ancillary_off_cycle_electric_power(0.0), design_heat_recovery_water_flow_rate(0.0), maximum_temperature_for_heat_recovery(80.0)
+    {
+    }
+
```

Comment:
The basic change here was to split getUnitarySystemInputData into 2 parts. A more lean getUnitarySystemInputData specific to UnitarySystem and it's report variables and a second function, processInputData, that can be used by other code (e.g., a getInput for DXSystem's CoilSystem model). Code duplication in the future should be minimized.

Reply:
:+1: